### DISCUSSION TITLE
Your PayPal Integration is due for an update

### DISCUSSION THREAD

**Discussion by Outlaw50** [OpenMage/magento-lts#3134] [Questions & Answers]
Hi All,

Recently switched from Magento 1.9 to OpenMage 20.0.20 due to host server upgrade and PHP upgrade, but now PayPal has come knocking.

Got more then 1 email so far saying:

----
Your PayPal Integration is due for an update

It looks like you’re using an older PayPal integration. Update now to offer Pay in 4, alternative payment methods, debit and credit cards, and our latest features. There’s no additional cost.

Which sends me here: https://www.paypal.com/au/webapps/mpp/campaigns/business/integration-update
---

Does OpenMage have the (or upcoming) update required and PayPal is just not seeing it or should I look into a plugin that supports the new PayPal stuff ?

Kind regards.
Kris.

**Comment by elidrissidev** [comment]
I don't think the Paypal integration got updated since Magento EOL. There are [tons of issues](https://github.com/OpenMage/magento-lts/issues?q=is%3Aissue+is%3Aopen+paypal) about Paypal and perhaps it can be looked into in the future if many people have the need for it.

**Comment by Outlaw50** [comment]
Looked like that when I went searching, but couldn't see anyone asking recently about it being supported and when I looked into the files not much changed from Magento to OpenMage - guess its best to look at a module (if any exist or are current)

Worst case ... another upgrade but that's a huge ordeal to do as its a platform change..

**Comment by pauldpauld** [comment]
There is this discussion/patch on "pay in four installments" here;
https://github.com/OpenMage/magento-lts/pull/1437
I don't know if that helps?
